scripts/setup-operators-41.sh

```diff
@@ -2,6 +2,47 @@
 set -x
 
 CI_OPERATOR_HUB_PROJECT=ci-operator-hub-project
+
+install_mongo_operator() {
```

Comment:
Good to have those, but tish will be dead code atm as 4.1 cluster has been disabled. You can remove any new changes making for 4.1 cluster.